src/util/importROAM.ts

```diff
@@ -0,0 +1,34 @@
+import { SimplePath } from '../types'
+import { State } from './initialState'
+import { importJSON } from './importJSON'
+import { Block } from '../action-creators/importText'
+import { ROAMChild, RoamNode } from 'roam'
+
+/**
+ * Recursively converts the roam children to blocks.
+ */
+const convertROAMChildrenToBlocks = (children: ROAMChild[]): Block[] => children.map(child => ({
+  scope: child.string,
+  children: child.children && child.children.length ? convertROAMChildrenToBlocks(child.children) : []
+}))
+
+/**
+ * Converts the ROAM JSON to an array of blocks.
+ */
+const convertROAMJSONToBlocks = (ROAM: RoamNode[]) => {
```

Comment:
Let's call this `roamJsonToBlocks`. The word `convert` is extraneous. All pure functions "convert` one thing to another.

Reply:
Good catch, thanks 👍

Reply:
@anmolarora1

Reply:
Sorry about this miss, @raineorshine . Would refactor it